### PR TITLE
Subdomain handling

### PR DESCRIPTION
**Pull request by scjody**
@chriddyp Please review.

### CODE REVIEW

**Comment by chriddyp** [comment]
deployed a test app: https://dashboardsly-pr-17.herokuapp.com/create

**Comment by chriddyp** [comment]
looks like there is a missing `/`: 
image

File: dashboardsly/src/js/actions/AppActions.js [hunk 1]
```diff
@@ -202,7 +202,7 @@ var AppActions = {
 
             request({
                 method: 'GET',
-                url: location.origin+'/dashboard?id='+dashboard_id
+                url: location.origin + CONFIG.ROOT_PATH + 'dashboard?id='+dashboard_id
```
Comment (chriddyp) [inline dashboardsly/src/js/actions/AppActions.js]: looks like we should add a `/` before `'dashboard'` here

File: dashboardsly/src/js/actions/AppActions.js [hunk 2]
```diff
@@ -147,7 +147,7 @@ var AppActions = {
             const username = AppStore.getState().username;
             const page = AppStore.getState().page;
             const apikey = AppStore.getState().apikey;
-            var url = location.protocol + '//' + window.location.host + '/files?username='+username+'&page='+page+'&apikey='+apikey;
+            var url = location.protocol + '//' + window.location.host + CONFIG.ROOT_PATH + 'files?username='+username+'&page='+page+'&apikey='+apikey;
```
Comment (chriddyp) [inline dashboardsly/src/js/actions/AppActions.js]: and a `/` before `'files'` here

File: dashboardsly/templates/base.html [hunk 3]
```diff
@@ -51,7 +51,7 @@
         ENV = {};
         ENV.mode = "{{mode}}";
     </script>
-    <script src="/static/js/bundle.js"></script>
+    <script src="{{ url_for('static', filename='js/bundle.js') }}"></script>
```
Comment (chriddyp) [inline dashboardsly/templates/base.html]: nice!!

File: dashboardsly/src/js/actions/AppActions.js (hunk 1 above)
Comment (scjody) [inline dashboardsly/src/js/actions/AppActions.js]: That should come from the default ROOT_PATH.  I'll investigate.

**Comment by chriddyp** [comment]
otherwise looks good!

**Comment by scjody** [comment]
@chriddyp Fixed the `/` issue and tested.  OK to merge?

**Comment by chriddyp** [comment]
:dancer:

**Pull request merged**